tests/pipeline/test_python_component.py

```diff
@@ -110,11 +112,19 @@ def transform(self, dataframe: pd.DataFrame) -> pd.DataFrame:
     )
     assert len(pipeline._graph.keys()) == 1 + 1
     assert pipeline._graph["AddN"]["dependencies"] == ["CreateData"]
+    pipeline._graph["AddN"]["operation"].operation_spec.to_json()
+
+    basename = "fndnt/fondant-base"
+    fondant_version = version("fondant")
+    python_version = sys.version_info
+    python_version = f"{python_version.major}.{python_version.minor}"
+    fondant_image_name = f"{basename}:{fondant_version}-python{python_version}"
```

Comment:
This you want to mock ? Tbh I think the way it is now is fine since it will nicely test all the python versions since we run tox with multiple versions.